styles/sizes.css

```diff
@@ -0,0 +1,49 @@
+.s-1 {
+    width: 1rem !important;
```

Comment:
using !important is not recommended unless absolutely necessary , might cause problems later.